### DISCUSSION TITLE
How stable is Quickemu when running WSL2 as the host?

### DISCUSSION THREAD

**Discussion by shawny43** [quickemu-project/quickemu#985] [Quickemu] [answered]
I tried to run Quickemu mac directly from WSL2 with Ubuntu but the mouse capture doesn't work and the window of the app broke somehow. There some tricks i can do?

**Comment by flexiondotorg** [comment, marked as answer]
Support for using WSL2 as the host is incomplete right now. It is known not to work reliably, and we welcome any contributions to help improve WSL2 support.
